debian/focal/foreman/rules

```diff
@@ -22,6 +22,7 @@ build:
 	/bin/cp config/dynflow/worker.yml.example config/dynflow/worker.yml
 	/bin/mkdir -p ./vendor/cache
 	cd ./vendor/cache && /usr/bin/gem fetch thor -v '1.1.0'
```

Comment:
Is this line still needed if we explicitly pin here?

Reply:
probably not, but I wanted to start small ;)